### PR TITLE
Renesas SCE port fix aes128-gcm crypt only usecase

### PR DESCRIPTION
**Pull request by miyazakh**
Fix aes128-gcm crypt only use-case
Update signed ca certification because ca-cert was updated

### CODE REVIEW

File: wolfcrypt/src/port/Renesas/renesas_sce_aes.c [hunk 1]
```diff
@@ -173,11 +173,19 @@ WOLFSSL_LOCAL int  wc_sce_AesGcmEncrypt(struct Aes* aes, byte* out,
...
-          iv_l = iv;
-          ivSz_l = ivSz;
+            } else {
+                XMEMCPY(&key_client_aes, &info->sce_wrapped_key_aes128,
+                    sizeof(sce_aes_wrapped_key_t));
+            }
+            iv_l = iv;
+            ivSz_l = ivSz;
+        } else {
+            WOLFSSL_MSG("AES kes for SCE is not set.");
```
Comment (cconlon) [inline wolfcrypt/src/port/Renesas/renesas_sce_aes.c:187]: Spelling - "kes" should be "key"

File: wolfcrypt/src/port/Renesas/renesas_sce_aes.c [hunk 2]
```diff
@@ -346,11 +354,19 @@ WOLFSSL_LOCAL int  wc_sce_AesGcmDecrypt(struct Aes* aes, byte* out,
...
+                XMEMCPY(&key_server_aes, &info->sce_wrapped_key_aes256,
+                    sizeof(sce_aes_wrapped_key_t));
+            } else {
+                XMEMCPY(&key_server_aes, &info->sce_wrapped_key_aes128,
+                    sizeof(sce_aes_wrapped_key_t));
+            }
             iv_l = iv;
             ivSz_l = ivSz;
+        } else {
+            WOLFSSL_MSG("AES kes for SCE is not set.");
```
Comment (cconlon) [inline wolfcrypt/src/port/Renesas/renesas_sce_aes.c:368]: Spelling - "kes" should be "key"

**Review by cconlon — CHANGES_REQUESTED** [review]

File: wolfcrypt/src/port/Renesas/renesas_sce_aes.c (hunk 1 above)
Comment (miyazakh) [reply wolfcrypt/src/port/Renesas/renesas_sce_aes.c:187]: Thanks. Fixed

File: wolfcrypt/src/port/Renesas/renesas_sce_aes.c (hunk 2 above)
Comment (miyazakh) [reply wolfcrypt/src/port/Renesas/renesas_sce_aes.c:368]: Thanks. Fixed

**Review by cconlon — APPROVED** [review]

**Pull request merged**